pom.xml

```diff
@@ -40,7 +40,7 @@
     <mockito.version>2.28.2</mockito.version>
     <slf4j.version>1.7.24</slf4j.version>
     <timestream.version>1.11.872</timestream.version>
```

Comment:
nit: can we update the SDK version to recent? 1.12.x ? - https://github.com/aws/aws-sdk-java/tags

Reply:
Suggesting to do a version upgrade not related to this change in a separate PR, for better trackability, so it is also easier to roll back if needed.

Reply:
Pls add a follow up tracking issue here  - https://github.com/awslabs/amazon-timestream-driver-jdbc/issues

Reply:
Added - https://github.com/awslabs/amazon-timestream-driver-jdbc/issues/24